packages/pigeon/run_tests.sh

```diff
@@ -129,7 +130,7 @@ if [ ! -f "ci/$JAVA_FORMATTER" ]; then
   curl -L https://github.com/google/google-java-format/releases/download/google-java-format-1.3/$JAVA_FORMATTER > "ci/$JAVA_FORMATTER"
 fi
 if [ ! -f "ci/$GOOGLE_CHECKS" ]; then
-  curl -L https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/$GOOGLE_CHECKS > "ci/$GOOGLE_CHECKS"
+  curl -L https://raw.githubusercontent.com/checkstyle/checkstyle/$GOOGLE_CHECKS_VERSION/src/main/resources/$GOOGLE_CHECKS > "ci/$GOOGLE_CHECKS"
```

Comment:
How is this related to the contributing guide? I'm not sure what this is based on the PR description.

Reply:
updated description, it's only related in the fact that this is the first PR that has run into the problem of drift in the CI script